packages/sanity/src/core/form/studio/inputResolver/fieldResolver.tsx

```diff
@@ -8,8 +8,16 @@ import {FormField, FormFieldSet} from '../../components'
 import {ChangeIndicator} from '../../../changeIndicators'
 import {getTypeChain} from './helpers'
 
-function PassThrough({children}: {children: React.ReactNode}) {
-  return <>{children}</>
+function PassThrough(field: FieldProps) {
```

Comment:
Maybe rename this to `BooleanField`? I think the idea was at some point that could be useful for other type of fields too, but it seems to be only used by boolean fields.

Reply:
Ok!